npm-init.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+
+const content = `{ 
+  "extends": "semistandard"
+}`
+fs.writeFileSync('.eslintrc.json', content);
+fs.mkdirSync('test');
+
+module.exports = {
+  name: prompt('name', basename || package.name),
+  version: '0.0.1',
+  description: prompt(s => s),
+  main: prompt('entry point', 'index.js', ep => fs.writeFileSync(ep, '')),
```

Comment:
Can we write the index.js file with a simple template like 'module.exports = exports = {};]\n` Just so it's non-empty and ends in a linefeed?